projects/swimlane/ngx-charts/src/lib/common/grid-panel-series.component.ts

```diff
@@ -11,11 +10,6 @@ interface GridPanel {
   y: number;
 }
 
-enum GridOrientation {
```

Comment:
Use `BarOrientation` instead in order to have not two `enum`s with the same values.

Reply:
There is already an `enum` called `Orientation` (see `projects/swimlane/ngx-charts/src/lib/common/types/orientation.enum.ts`).

Reply:
Ahh you're right.